CONTRIBUTING.md

```diff
@@ -50,12 +50,12 @@ first.
 
 ## Regression tests
 
-You are strongly encouraged to add regression tests. Regression tests are great,
+You are **strongly encouraged** to add regression tests. Regression tests are great,
 not least because they:
 
-* ensure that your contribution will never completely stop working,
+* Ensure that your contribution will never completely stop working.
```

Comment:
I personally believe that starting the bullet points with a lowercase letter makes sense, as it's like a continuation of the sentence - if you take the punctuation/markdown formatting away, you get "not least because they ensure[...]" which is easier on the eyes/more natural than "not least because they Ensure[...]"

Reply:
I see your point. I didn't know which was dramatically correct but I went and did a quick lookup and found this. The points you make are valid and grammatically correct. Neat, the more you know I guess!

https://www.imperial.ac.uk/brand-style-guide/writing/grammar/punctuation/bullet-points/